CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## v1.14.0 (15/11/2021)
+
+**Common**
+* **Work In Progress** Added LMDB library as a dependency. Currently not used, turned off.
+* Added handling of the `SSLException` error on Android as `IO_ERROR`.
+* Added handling of the `NSURLErrorCannotConnectToHost` error on iOS as `IO_ERROR`.
```

Comment:
Rather:
"Added proper handling of the ... error in the [Android / iOS] `olp::http::Network` implementation. Now it is mapped to `olp::http::ErrorCode::IO_ERROR` and subsequently handled as a retryable err´´or."